### DISCUSSION TITLE
Wayland Support? *solved*

### DISCUSSION THREAD

**Discussion by vonkraus** [cryinkfly/Autodesk-Fusion-360-for-Linux#221] [Q&A] [answered]
Good day!  Thank you for all the hard work you do.  Love being able to run fusion 360 on my Fedora unit.  Any way, do you have plans to build in wayland support?  I currently on can run it on x11, which is fine, but since the mentions of wayland replacing x11 possibly this year; I'm eager to know your thoughts!

Thanks again!
vonkraus

**Comment by cryinkfly** [comment, marked as answer]
@vonkraus Here can you find the answer:

- https://github.com/cryinkfly/Autodesk-Fusion-360-for-Linux/wiki/Documentation
- https://github.com/cryinkfly/Autodesk-Fusion-360-for-Linux/issues/47

**Reply by vonkraus** [reply]
Thank you!
